yt_dlp/extractor/youtube.py

```diff
@@ -238,6 +238,16 @@
         },
         'INNERTUBE_CONTEXT_CLIENT_NAME': 85
     },
+    # This client has 1080p non split stream videos
+    'media_connect_frontend': {
```

Comment:
as pukkandan mentioned on discord, `_` has special meaning in client names, so we need to change this to something `mediaconnect`